src/mgr/PyModule.cc

```diff
@@ -362,9 +362,11 @@ int PyModule::load(PyThreadState *pMainThreadState)
       const wchar_t *argv[] = {L"ceph-mgr"};
       PySys_SetArgv(1, (wchar_t**)argv);
       // Configure sys.path to include mgr_module_path
-      string paths = (g_conf().get_val<std::string>("mgr_module_path") + ':' +
+      auto mgr_module_path = g_conf().get_val<std::string>("mgr_module_path");
+      string paths = (mgr_module_path + ":" + mgr_module_path + "/" + pypackages_path + ':' +
```

Comment:
I was dubious about whether using `string`, `wstring` or `std::filesystem::path` (it's already used in some components), but then I saw that few lines below these changes, there's this `wstring sys_path`.

One fancy feature pf `std::filesystem::path` is that it overloaded `/` as concatenation operator (`path(mgr_module_path) / pypackages_path`)

What do you think?

Reply:
fs::path should be a better approach, I went for that.